DemocracyClub/WhoCanIVoteFor
# Feat/party emblems and descriptions

Phase 2 of the party pages overhaul. [Trello](https://trello.com/c/dVLjcV1A/3225-wcivf-political-party-pages).

I've ported over the `Emblem` and `PartyDescription` models from YNR and added these to the importer. Settled on adding them as separate methods called in the existing `import_parties` method so that it's easily refactorable later if we'd like to move it elsewhere. Again, the models have a couple of changes form their YNR counterparts, just small naming tweaks though so the new models don't tread on the toes of the existing ones.

Because the descriptions don't have a PK in the YNR payload I've put a unique constraint on the description and party together, and in light of this I set up some Integrity Error logging in case there's anything in the YNR data that doesn't meet this new criterion.

To test locally:
- run the migrations
- run `python manage.py import_parties`
- in your db you should have new tables for emblems and descriptions - run something like `select * from parties_partydescription/parties_partyemblem where party_id='party:52'` (using the Conservative party again because I know they have multiple emblems and multiple descriptions). Interestingly they don't have a default emblem but that's by-the-by. 

```[tasklist]
### PR Checklist

```
